java/client/src/org/openqa/selenium/AbstractCapabilities.java

```diff
@@ -17,6 +17,7 @@
 
 package org.openqa.selenium;
 
+import static org.openqa.selenium.remote.CapabilityType.LOGGING_PREFS;
```

Comment:
it fails in Travis on this line:
```
import static org.openqa.selenium.remote.CapabilityType.LOGGING_PREFS;
^
/home/travis/build/SeleniumHQ/selenium/java/client/src/org/openqa/selenium/By.java:-1: note: Some input files use or override a deprecated API.
```

But i can't figure our: what is deprecated here?